src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebar.tsx

```diff
@@ -200,7 +210,16 @@ export const ArtworkSidebar: React.FC<
 
       {!isUnlisted && !isSold && artworkEcommerceAvailable && (
         <>
-          <SidebarExpandable label="Shipping and taxes">
+          <SidebarExpandable
+            label={
+              <Flex flexDirection="column" justifyContent="flex-start">
+                <Text>Shipping and taxes</Text>
+                {globalArtsyShipping && artsyShippingEstimateEnabled && (
+                  <ArtsyShippingEstimate artwork={artwork} />
+                )}
+              </Flex>
+            }
+          >
             <ArtworkSidebarShippingInformationFragmentContainer
```

Comment:
As discussed in person - we are not hiding the actual shippingInfo line in expanded section of this view. We can definitely do it - just don't think it's adding anything to the test and believe keeping code less intermingled with the estimate test here is a good goal. Might still be worth confirming out assumption with Cami and Vivi.